config/supervisord.conf

```diff
@@ -57,7 +57,7 @@ autostart=true
 [program:scorebot]
 command=python -m rcon.scorebot
 environment=LOGGING_FILENAME=scorebot_%(ENV_SERVER_NUMBER)s.log
-startretries=10
+startretries=24
```

Comment:
could even put 1000

Reply:
I'm going to leave it as is so that it starts up on a restart, but doesn't cycle forever if people have actual errors.